portals/knowledge-management/src/App.vue

```diff
@@ -39,12 +39,24 @@ import AppLoading from '@/Components/AppLoading.vue';
 import MobileSidebar from '@/Components/MobileSidebar.vue';
 import DesktopSidebar from '@/Components/DesktopSidebar.vue';
 import Breadcrumbs from '@/Components/Breadcrumbs.vue';
+import determineIfUserIsAuthenticated from '@/Services/DetermineIfUserIsAuthenticated.js';
+import getAppContext from '@/Services/GetAppContext.js';
+import axios from '@/Globals/Axios.js';
+import { useTokenStore } from '@/Stores/token.js';
 
 const errorLoading = ref(false);
 const loading = ref(true);
 const showMobileMenu = ref(false);
 
+const userIsAuthenticated = ref(false);
+
 onMounted(async () => {
+    await axios.get('/sanctum/csrf-cookie');
```

Comment:
Will this work in the embeded version of the portal?

Reply:
Nope - good catch. This is not the correct endpoint.